docs/installation.rst

```diff
@@ -1,32 +1,71 @@
-============
+.. _installation:
+
 Installation
 ============
 
 This package requires NumPy, which you can install alone or as a part of SciPy.
-`Some Python distributions <http://www.scipy.org/install.html#scientific-python-distributions>`_ come with NumPy/SciPy pre-installed. For Python distributions
-without NumPy/SciPy, Windows/Mac users should install
-`pre-compiled binaries of NumPy/SciPy <http://www.scipy.org/scipylib/download.html#official-source-and-binary-releases>`_, and Linux users may have NumPy/SciPy
-available in `their repositories <http://www.scipy.org/scipylib/download.html#third-party-vendor-package-managers>`_.
+`Some Python distributions <https://www.scipy.org/install.html>`_
+come with NumPy/SciPy pre-installed. For Python distributions without
+NumPy/SciPy, Windows/Mac users should install
+`pre-compiled binaries of NumPy/SciPy <https://www.scipy.org/scipylib/download.html#official-source-and-binary-releases>`_, and Linux users may have
+NumPy/SciPy available in
+`their repositories <https://www.scipy.org/scipylib/download.html#third-party-vendor-package-managers>`_.
+
+When you have NumPy, you may use either PyPi or GitHub to install this package.
+
 
-When you have NumPy, install this package at the command line using
+.. _installation-pip:
+
+PyPi
+----
+This is the most straigforward option!  From the command line use
 ``pip`` [1]_::
 
     pip install apexpy
 
-This assumes that the same version of libgfortran is installed in the same location as when the pip wheel was built. If not, you may have trouble importing apexpy and you will have to build apexpy yourself using:
+This assumes that the same version of libgfortran is installed in the same
+location as when the pip wheel was built (in the event that a wheel is
+available). If not, you may have trouble importing apexpy.  In the event that
+you run into issues, you can get around this problem by using
+``pip`` [1]_::
+
+    pip install --global-option='build_ext' apexpy
 
-pip install --global-option='build_ext' apexpy
+or on Windows with the Mingw32 compiler, this command::
+
+    pip install --no-use-pep517 --global-option build_ext \
```

Comment:
Another way to fix things is to follow this: https://wiki.python.org/moin/WindowsCompilers#GCC_-_MinGW-w64_.28x86.2C_x64.29

I tested this and was able to simply use `pip install .` to install apexpy on windows. Thoughts?

Reply:
I hesitate to add setup.cfg commands that are only used for a particular setup, especially since mingw was such a pain in AppVeyor.

Reply:
I agree that we don't need to make changes to the `setup.cfg` in the `apexpy` repo. Perhaps we can revamp the Windows installation documentation in another release though.

Thinking more about this, I think it's best to just leave this documentation as it is and update it later so we don't delay this PR any longer. The documentation is already better now than it was before.